### PR TITLE
J bennet/release 2.2.0

### PR DESCRIPTION
**Pull request by j-bennet**
## Description

Prepare for a new release. Most importantly to fix https://github.com/dbcli/pgcli/issues/1121.

### CODE REVIEW

**Comment by lelit** [comment]
OOC, why the restriction? I'm using PT 3 since a while, and didn't notice any defect...

**Comment by j-bennet** [comment]
@lelit nobody did extensive testing of pgcli with 3.0, and it has incompatible changes according to @jonathanslenders, so I wanted to be on the safe side. Did you run unit tests / behave tests with 3.0? Does everything pass?

**Comment by lelit** [comment]
No, sorry: I browsed its changelog and didn't spot alerting changes... so I upgraded it.
Didn't have surprises, using it. But if Jonathan said so, the restriction is surely due.

File: setup.py [hunk 1]
```diff
@@ -13,10 +13,10 @@
 description = "CLI for Postgres Database. With auto-completion and syntax highlighting."
 
 install_requirements = [
-    "pgspecial>=1.11.5",
+    "pgspecial>=1.11.8",
     "click >= 4.1",
     "Pygments >= 2.0",  # Pygments has to be Capitalcased. WTF?
-    "prompt_toolkit>=2.0.6,<2.1.0",
+    "prompt_toolkit>=2.0.6,<3.0.0",
```
Comment (jonathanslenders) [inline setup.py]: Notice that this won't upgrade to version 3. it's ``<``, not ``<=``. Better use ``<3.1.0`` or ``4.0.0``.

Comment (lelit) [reply setup.py]: But that is intentional, given what Irina said, isn't it?

> it has incompatible changes according to @jonathanslenders, so I wanted to be on the safe side

Comment (lelit) [reply setup.py]: FWIW, I tried to execute the test suite using latest PT 3.0.2, and didn't notice any difference in the result, but I couldn't spend enough time to investigate on the failures (sorry about that)...
Given that nothing broke badly, neither in the tests, nor in my daily usage, hopefully the _incompatible changes_ are not impacting pgcli usage.

Comment (lelit) [reply setup.py]: To be clear, the same tests failed before and after upgrading PT.

Comment (j-bennet) [reply setup.py]: Yikes. We better make sure to fix whatever is broken then.

**Pull request merged**